src/main/java/edu/oakland/production/database/DatabaseGisInterfaceImplementation.java

```diff
@@ -0,0 +1,35 @@
+package edu.oakland.production.database;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.production.database.DatabaseGisManagerStub;
+
+public class DatabaseGisInterfaceImplementation {
+
+  DatabaseGisManager manager;
+  /**
+   * Creates an database interface to middleware
+   * to provide methods for managing GIS info.
+   *
+   * @param m an instance of the DatabaseGisManagerImplementation class.
+   */
+
```

Comment:
Remove the empty line between the Javadoc and the method; some IDEs have issues with that.
```suggestion
```